src/libraries/System.Formats.Tar/tests/TarTestsBase.cs

```diff
@@ -611,5 +612,22 @@ public static IEnumerable<object[]> GetPaxAndGnuTestCaseNames()
                 yield return new object[] { name };
             }
         }
+
+        public static void ExtractWithTarTool(string source, string destination)
+        {
+            Assert.True(!OperatingSystem.IsWindows(), "This method is unsupported on Windows.");
+
+            using Process tarToolProcess = new Process();
+            tarToolProcess.StartInfo.FileName = "/usr/bin/tar";
+            tarToolProcess.StartInfo.Arguments = $"-xf {source} -C {destination}";
+
+            tarToolProcess.StartInfo.UseShellExecute = false;
+            tarToolProcess.StartInfo.RedirectStandardOutput = true;
```

Comment:
This can lead to deadlocks:
"When the child process writes enough data to fill its redirected stream, it is dependent on the parent. The child process waits for the next write operation until the parent reads from the full stream or closes the stream. The deadlock condition results when the caller and child process wait for each other to complete an operation, and neither can continue."
Even though you don't care about the output, you probably want to add a call to tarToolProcess.StandardOutput.ReadToEnd() just before the WaitForExit call.